dts/bindings/led_strip/worldsemi,ws2812-rpi_pico-pio.yaml

```diff
@@ -50,11 +50,12 @@ child-binding:
   include: ws2812.yaml
 
   properties:
-    output-pin:
-      type: int
+    in-gpios:
+      type: phandle-array
       required: true
       description: |
-        Select the output pin.
+        GPIO phandle and specifier for the pin connected to the daisy
+        chain's input pin. Exactly one pin should be given.
```

Comment:
This doesn't really make sense, it it still the output pin from the MCU side, it should remain named as output. You wouldn't name the MISO pin on an SPI bus as "output-gpios" because it's an output from the sensor side.

Reply:
@nordicjm 
Thank you for comment.

I am also aware of the contradictions and will consider them as well.
It's a definition borrowed from ws2812-gpio, and also the original doesn't seem to be good.

https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/led_strip/ws2812_gpio.c#L239

This code uses `gpio_in` as output....

I think this is also included the scope for reconsideration.

I was looking at the review request for https://github.com/zephyrproject-rtos/zephyr/pull/67610.
I'm considering to create common-named gpio property between ws2812-gpio and ws2812-rpi-pico-pio for enabling to define common overlay.

I was interrupted and didn't pay attention to the details. Sorry for the confusion.

Reply:
I think a more common binding would probably be a good idea. With all of the different variations of the binding and drivers, you almost have to do something like what I did in #67610 did in order to get many boards supported.